telemetry/about-telemetry.adoc

```diff
@@ -0,0 +1,17 @@
+[id="about-telemetry"]
+= About Telemetry
+include::modules/common-attributes.adoc[]
+:context: telemetry
+
+toc::[]
+
+Telemetry collects anonymized aggregated information about:
+
+* the size of an OpenShift cluster
+* the health and status of OpenShift components
+* utilization of OpenShift components
```

Comment:
s/utilization/use

What's the difference between features and components?

What's the difference between this list and the list in the included module?